fast_depends/utils.py

```diff
@@ -81,9 +80,11 @@ def get_typed_signature(call: Callable[..., Any]) -> Tuple[inspect.Signature, An
 
     locals = collect_outer_stack_locals()
 
-    # If call is wrapped, __globals__ point to the wrapper's globals
-    # We need to get the globals of the wrapped function
-    globalns = sys.modules.get(call.__module__).__dict__
+    # We unwrap call to get the original unwrapped function
+    while hasattr(call, "__wrapped__"):
```

Comment:
Is it the same with `inspect.unwrap`?

Reply:
I think so. I didn't know about that. Since `unwrap` was added in 3.4, I'll change to that. 
Also, I forgot pydantic v1 didn't have `'model_rebuild'` yet. I'll fix this one also